scylladb/scylla
# docs: migrate the docs from the scylla-docs repo

This PR migrates the ScyllaDB end-user documentation from the [scylla-docs](https://github.com/scylladb/scylla-docs/) repository, according to the [migration plan](https://docs.google.com/document/d/15yBf39j15hgUVvjeuGR4MCbYeArqZrO1ir-z_1Urc6A/edit?usp=sharing). All the files are added to the `docs` subfolder.

 **This PR does not cover any content changes.**

How to test this PR:

1. Go to `scylla/docs`.
2. Run `make preview`. The docs should build without any warnings.
3. Open http://127.0.0.1:5500/ in your browser. You should see the documentation landing page:
![image](https://user-images.githubusercontent.com/37244380/177358869-af9f1b78-e528-4d0d-9479-cc69e25f3b67.png)
